contracts/scripts/v0.1/admin-aggregator/generate-aggregator-deployments.cjs

```diff
@@ -1,22 +1,53 @@
 const { getFormattedDate, loadJson, storeJson } = require('../utils.cjs')
 const { ethers } = require('hardhat')
+const path = require('path')
 
-const priceFeeds = []
+// ex. node ./scripts/v0.1/admin-aggregator/generate-aggregator-deployments.cjs --pairs '["usd-krw", "jpy-usd", "joy-usdc"]' --chain baobab
```

Comment:
Can you please mention this script with its example use case in README of contracts? It might be hard to discover these features only from code 🙏